src/datasets/arrow_dataset.py

```diff
@@ -3997,14 +4012,15 @@ def get_indices_from_mask_function(
     indices_mapping: Optional[Table] = None,
     *args,
     **fn_kwargs,
+    **tqdm_kwargs, 
```

Comment:
Having two `**kwargs` arguments is not allowed in python.

I think you can simply remove all mentions of `**kwargs` in this function, it doesn't need it